Mathlib/Topology/Sets/Opens.lean

```diff
@@ -336,6 +351,30 @@ theorem isCompactElement_iff (s : Opens α) :
     show ∀ i ∈ t, U i ≤ t.sup U
     exact fun i => Finset.le_sup
 
+/-- If `X` has a basis of compact opens and `f : X → S` is open, every
+compact open of `S` is the image of a compact open of `X`. -/
+lemma _root_.IsOpenMap.exists_opens_image_eq_of_isBasis (f : α → β) {B : Set (Opens α)}
+    (hB : IsBasis B) (hBc : ∀ U ∈ B, IsCompact U.1) (hfc : Continuous f) (h : IsOpenMap f)
```

Comment:
The spelling of "If `X` has a basis of compact opens"  should be `PrespectralSpace` (See `PrespectralSpace.of_isTopologicalBasis`).